extensions/azure-app-configuration/runtime/src/main/java/io/quarkiverse/azure/app/configuration/AzureAppConfigurationConfigSourceFactory.java

```diff
@@ -23,6 +24,8 @@
 public class AzureAppConfigurationConfigSourceFactory
         implements ConfigurableConfigSourceFactory<AzureAppConfigurationConfig> {
 
+    private static final String AZURE_QUARKUS_APP_CONFIGURATION = "az-qk-app-config";
```

Comment:
Do you think we need a class to hold all application ids just like [AzureSpringIdentifier](https://github.com/Azure/azure-sdk-for-java/blob/2c68397adca8db969f35d5787d5c906213d164bc/sdk/spring/spring-cloud-azure-core/src/main/java/com/azure/spring/cloud/core/implementation/util/AzureSpringIdentifier.java#L17C20-L17C41)?

Reply:
I like this idea from @backwind1233 . Please make it so.

Reply:
Good idea, addressed in 5d0833d.